woocommerce/woocommerce
# fix: notice button styles incorrect on WooCommerce.com subscriptions screen

### All Submissions:


### Changes proposed in this Pull Request:



Closes #21637.

### How to test the changes in this Pull Request:

1. Have at least one notice present
2. Navigate to the WooCommerce > Extensions > WooCommerce.com Subscriptions screen
3. Styles for the button(s) are consistent as other notice buttons

### Visual (Screenshot)
![image](https://user-images.githubusercontent.com/26460087/47600631-39c75080-d9e2-11e8-93a4-7c352773e466.png)

### Other information:




### Changelog entry

> Enter a short summary of all changes on this Pull Request. This will appear in the changelog if accepted.
